handlers/handlers.go

```diff
@@ -134,12 +134,16 @@ func (r RemoveHandler) ShouldHandle() bool {
 
 func (r RemoveHandler) Handle(m *monitors.Monitors) bool {
 	for _, monitor := range *m {
+        if len(monitor.EmptyDesktops()) == 1 {
```

Comment:
@cmschuetz One side effect of this is that an empty desktop could end up in the middle of the desktops.
I.E. where bold means not empty
"**1** **2** **3** 4"
// close window in 2, 2 is focused so `btops` removes 4
"**1** 2 **3**"

I suggest adding in a `MoveHandler`, and having the user configure if a configuration option to set `move-empty` to a value such as `first, last, keep`
A `MoveHandler` may eventually be used along with the `RenameHandler` to move a desktop to the specified position if it contains a window or has a name, i.e:
```
[[names.classified]]
"web" = ["Chromium", "Firefox"]
position = 0


[[names.classified]]
"" = ["Termite", "Alacritty", "URxvt"]
position = 1
```
This would share a similar `priority` metric as `RenameHandler` would (if I'm skimming the code correctly)